protostar/io/output.py

```diff
@@ -39,7 +62,28 @@ def __call__(self, message: Message):
                 f"JSON output is not supported for messages of type {type(message).__name__}."
             )
 
-        dictionary = message.format_dict()
-        json.dump(dictionary, sys.stdout, allow_nan=False, separators=(",", ":"))
+        obj = message.format_dict()
+        self._print(obj)
+
+    @contextmanager
+    def activity(self, message_template: Message) -> Generator[None, None, None]:
```

Comment:
I'm wondering if that's something that users would like to see in the output... I think the activity indicator thing is useful for having some UX thing making people aware that something is happening, whilst when having the json output i'd rather have the output in one plain json.

Reply:
I think I have a good reason for having these messages: measuring timing of the process in the tooling. With these messages, one is able to determine time cost of compilation minus Protostar booting time.

But nonetheless: I just copied what `cargo test` does in its JSON output. Maybe it doesn't really make sense here.

Another point of this PR was to move activity indicators to structured output framework. Maybe this command could simply skip JSON output as for now, like we did for `protostar format`?

Reply:
I think we can skip, since the output of the build is not that important - we know the preliminary contract names (and thus output names)

Reply:
done